packages/auth/amplify_auth_cognito/example/backend/lib/stack.ts

```diff
@@ -315,7 +333,7 @@ export class AuthIntegrationTestStack extends cdk.Stack {
       // Naming to match Amplify CLI, suffixed with a segment of the stack ID
       // https://github.com/aws-amplify/amplify-ci-support/blob/1abe7f7a1d75fa19675ad8ca17ab625a299b765e/src/integ_test_resources/flutter/amplify/cloudformation_template.yaml#L32
       bucketName: Fn.join("-", [
-        `amplify-test-${name.toLowerCase()}`,
+        name.toLowerCase(),
```

Comment:
Why remove "amplify-test"?

Reply:
I put it back - name was too long before but I forgot why we put amplify-test there to begin with